doc/vm-creation-example.md

```diff
@@ -0,0 +1,103 @@
+# Example- Allocating MAC Addresses to a VirtualMachine
+
+## Create a network-attachment-definition:
+
+The 'NetworkAttachmentDefinition' is used to setup the network attachment, i.e. secondary interface for the VirtualMachine.
+For more information see [Kubernetes Network Custom Resource Definition De-facto Standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit).
+This example will use a linux-bridge.
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+ name: br1
+spec:
+  config: '{
+    "cniVersion": "0.3.1",
+    "name": "br1",
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br1"
+        },
+        {
+            "type": "tuning"
+        }
+      ]
+    }'
+```
+
+**note** the tuning plugin changes the MAC Address after the main plugin was executed so
```

Comment:
That's a scary note, I think it rises more questions than it answers

Reply:
rephrased it, tell me what you think.

Reply:
Sounds better, thanks!